src/message_handler.md

```diff
@@ -1,6 +1,10 @@
 # Message handlers
 
-We've already seen that workers are basically components without widgets. In this chapter, we will talk about message handlers that are even simpler: like workers but without a model.
+Till now we have looked only on synchronously behaving applications. But in the real world this is not always the case. Often there is heavier calculation or REST API calls involved and we don't want our application to freeze. To solve this we will introduce a new type of component: a message handler, which is a basically a worker but without a model.
```

Comment:
This is not entirely correct, you can also use workers for asynchronous applications that don't freeze. But you can't handle more than one message at the time with workers. I don't think this is necessary here because there the whole "The motivation" paragraph and the following sections explaining the problem.

Reply:
Yes, probably makes sense to move in the motivator section.